src/main/resources/lib/nextjs/proxy.es6

```diff
@@ -120,14 +125,35 @@ function doRequest(originalReq, frontendRequestPath, xpSiteUrl, componentSubPath
 
         processNextjsSetCookieHeader(response);
 
-        nextjsCookies = getNextjsCookies();
+        if (!hadNextJsCookies || response.status >= 400) {
+            // Try reading the response message
+            let message = `Nextjs server did not return preview token [${response.status}]`;
+
+            if (response.message) {
+                message = `${message}: ${response.message}`;
+            } else {
+                try {
+                    const json = JSON.parse(response.body);
+                    if (json?.message) {
+                        message = `${message}: ${json.message}`;
+                    }
+                } catch (parseError) {
```

Comment:
Empty catch? Cant we print a json failure or just the raw json body then?

Reply:
It will still print the error and status on the line 144, so no need to print it here as well